src/ripple/rpc/handlers/GetCounts.h

```diff
@@ -0,0 +1,32 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
```

Comment:
Nit: Old copyright

Reply:
2012-2019? Most of the copyrights are out of date :)